### PR TITLE
revert zarith patches, introduce zarith-xen, adjust nocrypto

### PR DESCRIPTION
**Pull request by hannesm**

### CODE REVIEW

File: packages/zarith-xen/zarith-xen.1.3/opam [hunk 1]
```diff
@@ -0,0 +1,12 @@
+opam-version: "1"
+maintainer: "contact@ocamlpro.com"
+patches: ["xen_linkopts.patch"]
+build: [
+  ["./mirage-install.sh"]
+]
+depends: [
+  "mirage-xen"
```
Comment (avsm) [inline packages/zarith-xen/zarith-xen.1.3/opam]: This almost certainly needs a lower version on `mirage-xen`

File: packages/zarith-xen/zarith-xen.1.3/opam [hunk 2]
```diff
@@ -0,0 +1,12 @@
+opam-version: "1"
+maintainer: "contact@ocamlpro.com"
```
Comment (avsm) [inline packages/zarith-xen/zarith-xen.1.3/opam]: fix maintainer to mirageos-devel

**Comment by hannesm** [comment]
@avsm fixed

**Comment by hannesm** [comment]
pls restart job 5..

**Comment by samoht** [comment]
job 5 ignited.

**Comment by samoht** [comment]
Thanks!

**Pull request merged**